linkerd/app/outbound/src/target.rs

```diff
@@ -37,6 +37,9 @@ pub struct Endpoint<P> {
     pub concrete: Concrete<P>,
 }
 
+#[derive(Clone, Copy, Debug)]
+pub struct ShouldResolve;
```

Comment:
nit/TIOLI: i think this could have stayed a free fn and still worked with the filter interface?